DXRModules/DeusEx/Classes/DXRBingoCampaign.uc

```diff
@@ -0,0 +1,257 @@
+                }
+                dxr.flags.settings.starting_map = 140;
+                events.CreateBingoBoard();
+                ClearDataVaultImages();
+                break;
+            case "15_AREA51_BUNKER":
+                dxr.flags.settings.starting_map = 150;
+                events.CreateBingoBoard();
+                ClearDataVaultImages();
+                break;
+            case "15_AREA51_FINAL":
+                foreach AllActors(class'Switch1', button, 'destroy_generator_switch') {
+                    AddBingoEventBlocker(button, 'DXRando_Mission15_BingoCompleted');
+                    break;
+                }
+                foreach AllActors(class'#var(prefix)FlagTrigger', ft) {
+                    if (ft.event == 'Merge_helios_exit') {
+                        AddBingoEventBlocker(ft, 'DXRando_Mission15_BingoCompleted');
+                        break;
+                    }
+                }
+                break;
+            case "15_AREA51_PAGE":
+                foreach AllActors(class'Switch1', button, 'kill_page_switch') {
+                    AddBingoEventBlocker(button, 'DXRando_Mission15_BingoCompleted');
+                    break;
+                }
+                break;
+        }
+        break;
+    }
+}
+
+function AnyEntry()
+{
+    if (!dxr.flags.IsBingoCampaignMode()) return;
+    Super.AnyEntry();
+
+    switch (dxr.localURL) {
+        case "01_NYC_UNATCOISLAND":
+            GetConversation('GotoM02').AddFlagRef('DXRando_Mission01_BingoCompleted', true);
+            break;
+        case "02_NYC_BATTERYPARK":
+            GetConversation('BoatLeaving').AddFlagRef('DXRando_Mission02_BingoCompleted', true);
+            break;
+        case "02_NYC_WAREHOUSE":
+            GetConversation('JockExit').AddFlagRef('DXRando_Mission02_BingoCompleted', true);
+            break;
+        case "03_NYC_AIRFIELD":
+            GetConversation('M03JockLeave').AddFlagRef('DXRando_Mission03_BingoCompleted', true);
+            break;
+        case "05_NYC_UNATCOISLAND":
+            GetConversation('M05MeetJock').AddFlagRef('DXRando_Mission05_BingoCompleted', true);
+            break;
+        case "06_HONGKONG_WANCHAI_MARKET":
+            GetConversation('M06JockExit').AddFlagRef('DXRando_Mission06_BingoCompleted', true);
+            break;
+        case "08_NYC_STREET":
+            GetConversation('M08JockExit').AddFlagRef('DXRando_Mission08_BingoCompleted', true);
+            break;
+        case "09_NYC_GRAVEYARD":
+            GetConversation('M09JockLeave').AddFlagRef('DXRando_Mission09_BingoCompleted', true);
+            break;
+        case "10_PARIS_CHATEAU":
+            UpdateChateauInvisibleWall();
+            break;
+        case "11_PARIS_EVERETT":
+            GetConversation('TakeOff').AddFlagRef('DXRando_Mission11_BingoCompleted', true);
+            break;
+        // TODO: uncomment once Jock's event is getting changed after he appears
```

Comment:
You can just change the tag on the mapexit and insert the flagtrigger using the mapexit's original tag instead, then you don't need to worry about the event on the newly spawned copter